### ISSUE TITLE
Video preview doesn't works with a "poster" attribute

### ISSUE THREAD

**Issue by ArnaudLigny** [typora/typora-issues#5093]
**Describe the bug**
Video preview doesn't works with a "poster" attribute.

**To Reproduce**
1. Add a video tag to your content
2. See preview
3. Add a "poster" attribute
4. Preview disappear

**Expected behavior**
Video preview should still works with a "poster" attribute.

**Sample**
```html
<video controls poster="/videos/demo-cv-cecil-poster.png">
  <source src="/videos/demo-cv-cecil.mp4" type="video/mp4">
</video>
```

**Desktop (please complete the following information):**
 OS: Windows 10 64bit

**Typora Version**
1.1.5

**Comment by abnerlee** [comment]
fixed in 1.3.6

**Comment by ArnaudLigny** [comment]
Great, thanks a lot 👍
